pom.xml

```diff
@@ -50,17 +50,14 @@
       <artifactId>spring-web</artifactId>
     </dependency>
 
-    <!-- <dependency> <groupId>org.json</groupId> <artifactId>json</artifactId>
-      <version>20180130</version> </dependency> -->
-
     <dependency>
       <groupId>org.springframework.boot</groupId>
       <artifactId>spring-boot-starter-web</artifactId>
     </dependency>
 
     <dependency>
-      <groupId>com.jayway.jsonpath</groupId>
-      <artifactId>json-path</artifactId>
+      <groupId>org.springframework.boot</groupId>
```

Comment:
We seem to have a duplicate of this dependency now - please remove one of them.

Reply:
done

Reply:
I removed it completely actually